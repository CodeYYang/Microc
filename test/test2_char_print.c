int main(){
  char C='x';
  printf("%c",C);
  int i=2;
  printf("%d",i);
}