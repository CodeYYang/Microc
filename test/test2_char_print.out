24 19 0 5 25 13 0 120 12 13 11 23 15 -1 13 0 1 1 0 2 12 13 0 1 1 11 22 21 2